packages/back-end/src/api/experiments/updateExperiment.ts

```diff
@@ -21,6 +21,16 @@ export const updateExperiment = createApiRequestHandler(
       throw new Error("Could not find the experiment to update");
     }
 
+    // Validate projects - We can remove this validation when FeatureModel is migrated to BaseModel
+    if (req.body.project) {
+      const projects = await req.context.getProjects();
+      if (!projects.some((p) => p.id === req.body.project)) {
+        throw new Error(
+          `Project id ${req.body.project} is not a valid project.`
+        );
+      }
+    }
```

Comment:
Why not use `ensureProjectsExist` here too?  I know that function is meant for an array of projects, but it works fine for a single one too.  Just pass in `[req.body.project]`.  The only thing is the error message language is a little different, but I think that's fine.